src/test/full_functionality.spec.ts

```diff
@@ -379,6 +415,57 @@ describe("Mutations", () => {
             },
         })
     })
+
+    it("Argument is a int", () => {
+        const mutation = `
+        mutation {
+            getUsers(count: 1000) {
+                personal {
+                    count
+                }
+            }
+        }`
+        expect(graphQlQueryToJson(mutation)).toEqual({
+            mutation: {
+                getUsers: {
+                    __args: {
+                        count: 1000,
+                    },
+                    personal: {
+                        count: true,
+                    },
+                },
+            },
+        })
+    })
+
+    it("Argument is a variable", () => {
```

Comment:
This adds code extra code coverage for a line that was missing a test. This is nothing to do with the IntValue change. However, I did choose to use an IntValue in the test.

Adds test coverage for line 86 in index.ts